docs/pages/docs/ci/github-actions.mdx

```diff
@@ -71,15 +71,25 @@ Create file called `.github/workflows/ci.yml` in your repository with the follow
 
           steps:
             - name: Check out code
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
+
+            - name: Setup Cache
+              uses: actions/cache@v3
               with:
-                fetch-depth: 2
+                path: |
+                  node_modules/.cache/turbo
```

Comment:
This will grow unboundedly as `turbo` does not presently prune caches and it will use the alternative `restore-keys` to always end up with a hit.